prepare.sh

```diff
@@ -4,4 +4,44 @@ set -x
 
 yum -y install git podman
 
-git clone --single-branch --branch=master https://github.com/ceph/ceph-csi /opt/build
+while test $# -gt 0; do
```

Comment:
consider using `getopt` for parsing arguments, use `--longoptions help,ref:,workfdir:` for long names